frontend/src/components/orgSignUp/OrgSignUpForm.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import './OrgsSignUpForm.css';
+// import './OrgsSignUpForm.css';
```

Comment:
Should this be singular? Has been changed to singular in your first file (App.js).
We've used singular for our naming as well

Reply:
Yes we missed a few when we were amending the names to match backend, this has now been actioned.